core/config/docs/core.toml

```
@@ -291,6 +291,11 @@ ForceRedirect = false # Default
 # ListenIP specifies the IP to bind the HTTPS server to
 ListenIP = '0.0.0.0' # Default
 
+# Job Distributor stores the configuration for Job Distributor
+[JobDistributor]
+# NopFriendlyName is a friendly name set by the NOP to be shown in Job Distributor.
+NopFriendlyName = "MyDON-Production-01" # Example
```

This is a bad example. We don't want NOPs to put "DON" anywhere here. "Production" also doesn't make sense, every node run by NOPs is production :) 

Maybe
`NopDisplayName = "cre-workflow-1-chainlinklabs"`

(this is assuming that we want to include both the name of the NOP as well as the node. Maybe it's better to split those two...?

---

I think we can drop the `Nop` prefix too.
```suggestion
DisplayName = "cre-workflow-1-chainlinklabs" # Example
```

---

DisplayName seems better indeed, tell me what you think @cedric-cordenier @jmank88 @bolekk 
https://github.com/smartcontractkit/chainlink/pull/18606/commits/8f97554681a9a6a207a588a6160b5dccd0882457